### PR TITLE
Improve Bottles desktop entry comment

### PR DESCRIPTION
**Pull request by fries1234**
# Description
This PR improves the Bottles desktop file comment to make more sense.

## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### CODE REVIEW

**Comment by jannuary** [comment]
Thank you for your contribution! 

The app summary displayed on Flathub is different - it'd be nice to match them.

**Comment by fries1234** [comment]
looks like the summary on flathub is "Run Windows software"
image
should i change it to Easily manage Wine Prefixes or should i change the desktop comment to "Run Windows software"?

**Comment by mirkobrombin** [comment]
> looks like the summary on flathub is "Run Windows software"
> image
> should i change it to Easily manage Wine Prefixes or should i change the desktop comment to "Run Windows software"?

Some users may not know what a wineprefix is

**Comment by jannuary** [comment]
> should i change it to Easily manage Wine Prefixes or should i change the desktop comment to "Run Windows software"? 

The latter - it's much more clearer.

**Comment by fries1234** [comment]
oh ok :+1:

**Comment by fries1234** [comment]
lol wrong branch of my fork :pensive:

**Comment by A6GibKm** [comment]
There are multiple app sumaries that should match:

- shortdesc in the doap file
- Comment in the desktop file
- summary in the appdate
- comments in AdwAboutWindow

**Comment by TheEvilSkeleton** [comment]
Honestly, we should have some meson variable that automatically inserts the description without needing to modify several files.

**Comment by jannuary** [comment]
It's not somethings that's redacted frequently - it'd likely be harder to build and maintain this system than just edit it multiple times

**Comment by TheEvilSkeleton** [comment]
Can you elaborate on that? I've never done this so I'm in no position to argue about it. It only makes sense to me that automating the "boring" parts would help us maintain it.

**Comment by eli-schwartz** [comment]
Automating the boring parts, implies that changing the description is something that gets changed often enough to be worth automating.

And if you do automate it, then you pay the cost of additional file I/O at build time, which isn't intolerable but it also seems... wasted?

File: bottles/frontend/ui/about.blp [hunk 1]
```diff
@@ -4,6 +4,7 @@ using Gtk 4.0;
   application-name: "Bottles";
   modal: true;
   version: "2022.9.28.1";
+  comments: "Run Windows Software";
```
Comment (TheEvilSkeleton) [inline bottles/frontend/ui/about.blp:7]: This is discouraged by GNOME developers. https://matrix.to/#/!hwiGbsdSTZIwSRfybq:matrix.org/$16602639644946QoBIj:gnome.org?via=matrix.org&via=gnome.org&via=privacytools.io


```suggestion
```

**Review by TheEvilSkeleton — CHANGES_REQUESTED** [review]

File: bottles/frontend/ui/about.blp (hunk 1 above)
Comment (fries1234) [reply bottles/frontend/ui/about.blp]: ok i removed it do you have any more things you think i should do with the pr?

Comment (TheEvilSkeleton) [reply bottles/frontend/ui/about.blp]: Now we just need to test it and see if the change causes issues or not.

**Review by TheEvilSkeleton — APPROVED** [review]

**Pull request merged**